blueman/gui/manager/ManagerProgressbar.py

```diff
@@ -67,15 +69,20 @@ def __init__(self, blueman: "Blueman", cancellable: bool = True, text: str = _("
 
         ManagerProgressbar.__instances__.append(self)
 
+    def _get_window(self) -> Gdk.Window:
+        assert self.Blueman.window is not None
+        window = self.Blueman.window.get_window()
+        assert window is not None
+        return window
+
     def _on_enter(self, _evbox: Gtk.EventBox, _event: Gdk.Event) -> bool:
         c = Gdk.Cursor.new(Gdk.CursorType.HAND2)
-        assert self.Blueman.window is not None
-        self.Blueman.window.get_window().set_cursor(c)
+        self._get_window().set_cursor(c)
         return False
 
     def _on_leave(self, _evbox: Gtk.EventBox, _event: Gdk.Event) -> bool:
         assert self.Blueman.window is not None
-        self.Blueman.window.get_window().set_cursor(None)
+        self._get_window().set_cursor(None)
```

Comment:
If we are going to add a helper function let's move it to Blueman and use it everywhere we use Blueman.window probably getting rid of a few asserts in the process.